icicle-jedis/src/it/scala/com/intenthq/icicle/JedisIcicleIntegrationSpec.scala

```diff
@@ -1,15 +1,48 @@
 package com.intenthq.icicle
 
+import java.util
+
+import com.intenthq.icicle.redis.RoundRobinRedisPool
 import org.specs2.mutable._
 
-class JedisIcicleIntegrationSpec extends Specification {
+object JedisIcicleIntegrationSpec extends Specification {
   "constructor" should {
     "sets the correct host and port if a valid host and port is passed" in {
       val underTest = new JedisIcicle("localhost:6379")
       val jedis = underTest.getJedisPool.getResource
 
-      (jedis.getClient.getHost must_== "localhost") and
-        (jedis.getClient.getPort must_== 6379)
+      try {
+        (jedis.getClient.getHost must_== "localhost") and
+          (jedis.getClient.getPort must_== 6379)
+      } finally {
+        jedis.close()
+      }
     }
   }
+
+  "works with a real ID generator" in {
```

Comment:
This test here is one we never had before, but also doubles as a test to ensure that JARs that depend on `icicle-core` can still see the Lua script. I tested before and after the change and got the expected failure and success.